cfn-resources/private-endpoint/cmd/resource/steps/awsvpcendpoint/aws_vpc_endpoint.go

```diff
@@ -2,61 +2,95 @@ package awsvpcendpoint
+	VpcID               string
+	SubnetID            string
+	InterfaceEndpointID *string
+}
+
+type AwsPrivateEndpointOutput struct {
+	VpcID               string
+	SubnetID            string
+	InterfaceEndpointID string
+}
+
+func Create(req handler.Request, endpointServiceName string, region string, privateEndpointInputs []AwsPrivateEndpointInput) ([]AwsPrivateEndpointOutput, *handler.ProgressEvent) {
 	svc := newEc2Client(region, req)
 
 	vcpType := "Interface"
 
-	connection := ec2.CreateVpcEndpointInput{
-		VpcId:           &vpcID,
-		ServiceName:     &peCon.EndpointServiceName,
-		VpcEndpointType: &vcpType,
-		SubnetIds:       []*string{&subnetID},
-	}
+	subnetIds := make([]AwsPrivateEndpointOutput, len(privateEndpointInputs))
 
-	vpcE, err := svc.CreateVpcEndpoint(&connection)
-	if err != nil {
-		fpe := progress_events.GetFailedEventByCode(fmt.Sprintf("Error creating vcp Endpoint: %s", err.Error()),
-			cloudformation.HandlerErrorCodeGeneralServiceException)
-		return nil, &fpe
+	log2.Print("Entered Point 6")
+	for i, pe := range privateEndpointInputs {
+		log2.Print("Entered Point 7")
+		connection := ec2.CreateVpcEndpointInput{
+			VpcId:           &pe.VpcID,
+			ServiceName:     &endpointServiceName,
+			VpcEndpointType: &vcpType,
+			SubnetIds:       []*string{&pe.SubnetID},
+		}
+		log2.Print("Entered Point 8")
+		vpcE, err := svc.CreateVpcEndpoint(&connection)
+		if err != nil {
+			fpe := progress_events.GetFailedEventByCode(fmt.Sprintf("Error creating vcp Endpoint: %s", err.Error()),
+				cloudformation.HandlerErrorCodeGeneralServiceException)
+			return nil, &fpe
+		}
+
+		log2.Print("Entered Point 9")
+		subnetIds[i] = AwsPrivateEndpointOutput{
+			VpcID:               pe.VpcID,
+			SubnetID:            pe.SubnetID,
+			InterfaceEndpointID: *vpcE.VpcEndpoint.VpcEndpointId,
+		}
 	}
 
-	return vpcE.VpcEndpoint.VpcEndpointId, nil
+	log2.Print("Entered Point 10")
+	return subnetIds, nil
 }
 
-func Delete(req handler.Request, interfaceEndpoints []string, region string) (*ec2.DeleteVpcEndpointsOutput, *handler.ProgressEvent) {
+func Delete(req handler.Request, interfaceEndpoints []string, region string) *handler.ProgressEvent {
+	log2.Print("Entered Point 13.1")
 	svc := newEc2Client(region, req)
 
 	vpcEndpointIds := make([]*string, 0)
 
 	for i := range interfaceEndpoints {
 		vpcEndpointIds = append(vpcEndpointIds, &interfaceEndpoints[i])
 	}
-
+	log2.Print("Entered Point 13.2")
 	connection := ec2.DeleteVpcEndpointsInput{
 		DryRun:         nil,
 		VpcEndpointIds: vpcEndpointIds,
 	}
-
-	vpcE, err := svc.DeleteVpcEndpoints(&connection)
+	log2.Print("Entered Point 13.3")
+	_, err := svc.DeleteVpcEndpoints(&connection)
+	log2.Print("Entered Point 13.4")
 	if err != nil {
 		fpe := progress_events.GetFailedEventByCode(fmt.Sprintf("Error deleting vcp Endpoint: %s", err.Error()),
 			cloudformation.HandlerErrorCodeGeneralServiceException)
-		return nil, &fpe
+		return &fpe
 	}
 
-	return vpcE, nil
+	log2.Print("Entered Point 13.5")
```

Comment:
Can you explain the idea behind the logging here, or was it just used for debugging and left in accidentally?